endpoints/openrtb2/auction.go

```diff
@@ -1795,3 +1802,18 @@ func checkIfAppRequest(request []byte) (bool, error) {
 	}
 	return false, nil
 }
+
+func (deps *endpointDeps) setIntegrationType(req *openrtb_ext.RequestWrapper) error {
+	reqExt, err := req.GetRequestExt()
+	if err != nil {
+		return err
+	}
+	reqPrebid := reqExt.GetPrebid()
+	if reqPrebid == nil {
```

Comment:
What if `reqPrebid` is defined but `integration` was not specified? Shouldn't we apply the default integration type in that case?

Reply:
Yeah this is a good catch, I'll update this